literature.bib

```diff
@@ -83,6 +98,57 @@ @inproceedings{STS:VaMoS20
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{PAP+:ICST19,
+  title={Uniform sampling of sat solutions for configurable systems: Are we there yet?},
+  author={Plazar, Quentin and Acher, Mathieu and Perrouin, Gilles and Devroey, Xavier and Cordy, Maxime},
+  booktitle={2019 12th IEEE Conference on Software Testing, Validation and Verification (ICST)},
+  pages={240--251},
+  month={April},
+  year={2019},
+  doi={10.1109/ICST.2019.00032},
+  sampling-tags = {name=Uniform sampling, testing, non-functional properties, feature model, available tool, open-source tool, implementation artifacts, sampling efficiency, effectiveness, uses QuickSampler/UniGen},
+  organization={IEEE},
+  address ={Piscataway} 
+}
+
+@article{HNA+:ESE19,
+  title={Test them all, is it worth it? Assessing configuration sampling on the JHipster Web development stack},
+  author={Axel Halin, Alexandre Nuttinck and Mathieu Acher and Xavier Devroey and Gilles Perrouin and Benoit Baudry},
+  journal={Empirical Software Engineering},
+  volume={24},
+  number={2},
+  pages={674--717},
+  month={JUL},
+  sampling-tags = {evaluation, feature model, implementation artifacts, testing efficiency, effectiveness, testing, fault/failure efficiency, compares t-wise(SPLCAT)/one-disabled/one-enabled/most-enabled-disabled/dissimilarity(PLEDGE)/random},
```

Comment:
Already classified as HNA+:TR17

Make sure the classification is identical and contact me if you aim to change it.